java/src/main/java/io/opentracing/grpc/OperationNameConstructor.java

```diff
@@ -0,0 +1,13 @@
+package io.opentracing.contrib.grpc;
+
+public interface OperationNameConstructor {
+
+    public static OperationNameConstructor NOOP = new OperationNameConstructor() {
+        @Override
+        public String constructOperationName(String extractedName) {
+            return extractedName;
+        }
+    };
+
+    public String constructOperationName(String extractedName);
```

Comment:
I think it would be better to pass in `MethodDescriptor<ReqT, RespT> methodDesc` as a parameter here... gives more flexibility to the implementor. Would that work?